### ISSUE TITLE
Configurable command casing

### ISSUE THREAD

**Issue by taylor-peterson** [richq/cmake-lint#19]
It'd be nice to be able to detect/enforce consistent casing (e.g. always SET instead of set or vice versa).

**Comment by richq** [comment]
Yep this seems like a good idea. Right now it just checks for mixing the 2 types.
